### PR TITLE
Added a link to the TextVectorization API Reference Page.

### PR DESCRIPTION
**Pull request by Neeshamraghav012**
I have added a URL of the TextVectorization API in the StringLookup Layer page so that users can easily look at and experiment with the API.

### CODE REVIEW

File: keras/layers/preprocessing/string_lookup.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ class StringLookup(index_lookup.IndexLookup):
     This layer translates a set of arbitrary strings into integer output via a
     table-based vocabulary lookup. This layer will perform no splitting or
     transformation of input strings. For a layer than can split and tokenize
-    natural language, see the `TextVectorization` layer.
+    natural language, see the [TextVectorization](https://keras.io/api/layers/preprocessing_layers/text/text_vectorization/#textvectorization-class) layer.
```
Comment (qlzh727) [inline keras/layers/preprocessing/string_lookup.py:38]: I think you only need to change this to `tf.keras.layers.TextVectorization`, and our doc parse will convert it to a link to the TextVectorization API page. It doesn't necessary to put the explicit URL here.

**Review by qlzh727 — CHANGES_REQUESTED** [review]

File: keras/layers/preprocessing/string_lookup.py (hunk 1 above)
Comment (Neeshamraghav012) [reply keras/layers/preprocessing/string_lookup.py:38]: I tried that but it is not behaving like a like. That's why I added the explicit URL.

Comment (qlzh727) [reply keras/layers/preprocessing/string_lookup.py:38]: If you check https://www.tensorflow.org/api_docs/python/tf/keras/layers/TextVectorization and the corresponding source code at https://github.com/keras-team/keras/blob/e6784e4302c7b8cd116b74a784f4b78d60e83c26/keras/layers/preprocessing/text_vectorization.py#L64. The `tf.keras.layers.StringLookup` is rendered as a link.

**Comment by Neeshamraghav012** [comment]
I have done the required changes.

**Review by qlzh727 — APPROVED** [review]

**Pull request merged**